cf-execd/cf-execd.c

```diff
@@ -424,6 +424,17 @@ void ThisAgentInit(void)
     umask(077);
 }
 
+// Return 2 to 62 seconds depending on how much time is left until the next
+// minute starts. Since cf-execd wakes up "every minute" to evaluate its
+// schedule, we want to do this at the start of the minute, to avoid
+// accidentally skipping any runs if things are slow.
+static inline time_t GetPulseTime()
+{
+    const time_t current_second = (time(NULL) % 60);
+    const time_t remaining_seconds = 60 - current_second;
+    return remaining_seconds + 2;
```

Comment:
Why `+2`? Please add comment :)

Reply:
again, change these in the master PR and then re-cherry-pick right? :)